runtime/binding/python/README.md

```diff
@@ -47,15 +47,42 @@ The parameters of `extract_embedding_wav`
 import sys
 import wespeakerruntime as wespeaker
 wav_scp = sys.argv[1]
-inference = wespeaker.Inference(lang='chs')
-inference.extract_embedding(wav_scp, 'embed.ark')
+speaker = wespeaker.Speaker(lang='chs')
+speaker.extract_embedding(wav_scp, 'embed.ark')
 ```
 
-The parameters of `extract_embedding`
+The parameters of `extract_embedding`:
 - `wav_path` (str): the path of wav
 - `embed_ark` (str): the path of `$ouput`.ark
 - `resample_rate` (int): resampling rate. Default: 16000
 
+### Extract embedding from feature(fbank)
```

Comment:
The readme should be refactored if `extract_embedding` is kept as ndarray interface.

Here is the recommended order the outline.
1.  Extract speaker embedding
2.  Compute speaker simliarity
3.  Optional, Extract speaker embedding with fbank feature
4. Optional, Extract speaker embedding with kaldi IO